.chloggen/k8s-cluster-receiver-clusterresourcequota.yaml

```diff
@@ -0,0 +1,21 @@
+
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: "k8sclusterreceiver"
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Change k8s.clusterresourcequota metrics to use mdatagen"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [10553, 4367]
```

Comment:
Not sure if we are contributing to https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/10553 in this PR

Reply:
Good point, I removed it from the changelog and PR description.